ops/deploy-ccs.groovy

```diff
@@ -120,9 +186,53 @@ def buildAppAmis(AmiIds amiIds, AppBuildResults appBuildResults) {
  * @param appBuildResults (not used in the CCS environment; this stuff is all baked into the AMIs there, instead)
  * @throws RuntimeException An exception will be bubbled up if the deploy tooling returns a non-zero exit code.
  */
-def deploy(String envId, AmiIds amiIds, AppBuildResults appBuildResults) {
-	echo 'Deploy to the CCS environment is not yet implemented.'
-	throw new UnsupportedOperationException('Deploy to the CCS environment is not yet implemented.')
+def deploy(String environmentId, AmiIds amiIds, AppBuildResults appBuildResults) {
+	def env = normalizeEnvironmentId(environmentId)
+		dir("${workspace}/ops/terraform/env/${env}/stateless") {
+	
+		// Debug output terraform version 
+		sh "/usr/bin/terraform --version"
+		
+		// Initilize terraform 
+		sh "/usr/bin/terraform init"
+		
+		// Gathering terraform plan 
```

Comment:
https://github.com/CMSgov/beneficiary-fhir-data/blob/18e5c39cc55d62ccfef42fc69f61625193a69d4e/ops/deploy-ccs.groovy#L199-L211

My 2c, but I think it's potentially safer and more auditable to write the plan and read the apply from an `-out` file.

Reply:
good suggestion, done.